File: TUM_Lens/app/src/main/java/com/maxjokel/lens/helpers/FreezeCallback.java
package com.maxjokel.lens.helpers;

import android.graphics.Bitmap;

// new callback interface, to react to changes in 'viewFinder.java'
// [reference: https://stackoverflow.com/a/18054865]
public interface FreezeCallback {

    void onFrozenBitmap(Bitmap b);

}

File: TUM_Lens/app/src/main/java/com/maxjokel/lens/helpers/FreezeCallback.kt
package com.maxjokel.lens.helpers

import android.graphics.Bitmap

// new callback interface, to react to changes in 'viewFinder.java'
// [reference: https://stackoverflow.com/a/18054865]
interface FreezeCallback {
    fun onFrozenBitmap(b: Bitmap?)
}